packages/site/pages/components/actionmenu.js

```diff
@@ -271,10 +272,10 @@ export default _ => (
               'origin',
               PropTypes.union(ActionMenu.origins),
               null,
-              <code>topLeft</code>,
+              null,
               <span>
-                orientation (from <code>ActionMenu.origins</code>) of nested
-                menus
+                override orientation (from <code>ActionMenu.origins</code>) of
```

Comment:
`origin` controls orientation.  But if we remove the default, I think we leave open the question of what the *default* behavior is.  I'd ensure topLeft as the default origin.

Reply:
It's passed in via context on items no need for a default.

Reply:
So the default is whatever you set on the Menu component. I could kill this override all together. I just figured someone will wan to do it.

Reply:
The default is topLeft (from what I saw in your latest PR https://github.com/pluralsight/design-system/pull/1105).  I think we should keep that exposed as a defaultProps or at least in documentation.  This would serve the user of ActionMenu in learning what the observed default is called, helping him conceptualize what "topLeft" must mean and help him from there extrapolate what the others might mean.